src/ucm/util/sys.c

```diff
@@ -191,7 +191,8 @@ void ucm_parse_proc_self_maps(ucm_proc_maps_cb_t cb, void *arg)
             buffer = ucm_orig_mremap(buffer, buffer_size, buffer_size * 2,
                                      MREMAP_MAYMOVE, NULL);
             if (buffer == MAP_FAILED) {
-                ucm_fatal("failed to allocate maps buffer(size=%zu)", buffer_size);
+                ucm_fatal("failed to allocate maps buffer(size=%zu): %m",
```

Comment:
maybe print full params:
"failed to reallocate buffer for reading proc/self/maps from %p/%zu to size %zu"